README.md

```diff
@@ -49,23 +62,58 @@ datasets. Each of these subfolders are little project. They contain code and a
+|*(Ovid)*         |               |                |               |               |
+| *Pubmed*        | Not supported |                |Not supported  |               |
+| *Scopus*        | Supported     |                |Supported      |               |
+|*Web of*         | Not supported |Not supported   |               |               |
+|*Science*        |               |                |               |               |
+| **Systematic Review Software**|||||                                                   
+| *Abstrackr*     | Supported     |                | Supported     |               |
+| *Covidence*\*   | Supported     |                | Supported     |               |
+| *Distiller*     |Not supported  |                | Supported\**  |Supported\**   |
+|*EPPI-reviewer*  | Supported     |                |               |Not supported  |
+| *Rayyan*        | Not supported |                | Supported     |               |
+|*Robotreviewer*\***  |||||
 
-### RIS files
+- Supported: The data can be exported from the software and imported in ASReview LAB using this extension.
+- Not supported: The exported data can not be imported in ASReview LAB using this extension.
+- (empty): The data cannot be exported from the software using this extension.
 
+\* When using Covidence it is possible to export articles in .ris formats for different citation managers,
+such as Endnote, Mendeley, Refworks and Zotero. All of these are compatible with ASReview LAB.
+\** When exporting from Distiller set the ``sort references by`` to ``Authors``. Then the data can be
+imported in ASReview LAB.
+\*** Robotreviewer does not provide exports suitable for ASReview LAB, since it supports evidence synthesis.
+
+### Format of data without ASReview LAB
+If you would like to share your data without having used ASReview LAB for the screening of your records, or because you have done the screening manually, please make sure the datafile is in the right format. Two examples can be found at the bottom of the page.
+
+#### RIS files
 [RIS files][52] are used by
 digital libraries, like IEEE Xplore, Scopus and ScienceDirect. Citation
 managers Mendeley and EndNote support the RIS format as well. For simulation,
 we use an additional RIS tag with the letters `LI` (Label included).
 
-### CSV files
-
-For CSV files, the software accepts a set of predetermined labels in line with
-the ones used in RIS files. The most commonly used ones are: "id", "authors", "date", "title", "keywords" and "abstract". To indicate labelling decisions, one can use "included" or "label\_included". 
+#### Tabular datasets
+Extensions .csv, .xlsx, and .xls. CSV files should be comma separated and UTF-8 encoded. For CSV files, the software accepts a set of predetermined labels in line with the ones used in RIS files: "id", "authors", "date", "title", "keywords" and "abstract". To indicate labelling decisions, one can use "included" or "label\_included".
```

Comment:
Unless something has changed, id/authors/data do not have any real meaning in the context of the simulation mode (they are not used). If there is a column called "record_id", then it will be used as an index. There are a few more supported column names, but perhaps it is better to not confuse the reader with them.